### DISCUSSION TITLE
How can I achieve a clipPath with an svg path in Pixi?

### DISCUSSION THREAD

**Discussion by tansanDOTeth** [pixijs/pixijs#9065] [Q&A]
I'm learning Pixi.js right now (latest version v7.1.0), and I am trying to figure out how I can create the equivalent of this SVG. I have a clipPath which I essentially a mask. However, how can I draw a shape given the SVG path and it's d parameter? is there some SVG equivalent where I can just pass in the d value? or do I have to think about it differently? 

```
<svg viewBox="0 0 100 100">
  <clipPath id="myClip">
    <!--
      Everything outside the circle will be
      clipped and therefore invisible.
    -->
    <circle cx="40" cy="35" r="35" />
  </clipPath>

  <!-- The original black heart, for reference -->
  <path
    id="heart"
    d="M10,30 A20,20,0,0,1,50,30 A20,20,0,0,1,90,30 Q90,60,50,90 Q10,60,10,30 Z" />

  <!--
    Only the portion of the red heart
    inside the clip circle is visible.
  -->
  <use clip-path="url(#myClip)" href="#heart" fill="red" />
</svg>
```

**Comment by SuperSodaSea** [comment]
Maybe you can try to use packages like [`svg-pathdata`](https://github.com/nfroidure/svg-pathdata) to parse the d parameter, and convert the result to `PIXI.Graphics`' command by yourself.

**Reply by tansanDOTeth** [reply]
This seems like what I am looking for. I'll try this out! Thank you!
